etc/helm/pachyderm/templates/pgbouncer/deployment.yaml

```diff
@@ -79,6 +79,23 @@ spec:
         image: bitnami/pgbouncer:1.16.1-debian-10-r82
         imagePullPolicy: IfNotPresent
         name: pg-bouncer
+        livenessProbe:
+          failureThreshold: 3
+          exec:
+            command:
+              - "/bin/sh"
+              - "-c"
+              - psql postgresql://$POSTGRESQL_USERNAME:$POSTGRESQL_PASSWORD@127.0.0.1:$PGBOUNCER_PORT/ --tuples-only --command="SELECT 1;" | grep -q "1"
```

Comment:
I like that we're doing a thorough check.  I guess when this consistently returns "too many open connections to database", we'll restart, and then nothing will work for a while.  But maybe some of those dropped database connections were leaked and that's actually a good thing?

Reply:
yeah, i can test it on one of @BOsterbuhr or @dgeorg42 load testing setups to see how it behaves first